search/collector/topn.go

```diff
@@ -90,6 +91,19 @@ func NewTopNCollectorAfter(size int, sort search.SortOrder, after []string) *Top
 	rv.searchAfter = &search.DocumentMatch{
 		Sort: after,
 	}
+
+	for pos, ss := range sort {
+		if ss.RequiresDocID() {
+			rv.searchAfter.ID = after[pos]
+			continue
```

Comment:
oh, one thing -- perhaps get rid of this 'continue', just in case the ss object has both RequiresDocID() of true and RequiresScoring() of true?

Reply:
As of today, SortScore and SortDocID have these properties orthogonal or mutually exclusive. But perhaps in future if we have a new sort type, this could be problem then.